x/evm/types/chain_config.go

```diff
@@ -40,8 +40,7 @@ type ChainConfig struct {
 	IstanbulBlock       sdk.Int `json:"istanbul_block" yaml:"istanbul_block"`             // Istanbul switch block (< 0 no fork, 0 = already on istanbul)
 	MuirGlacierBlock    sdk.Int `json:"muir_glacier_block" yaml:"muir_glacier_block"`     // Eip-2384 (bomb delay) switch block (< 0 no fork, 0 = already activated)
 
-	YoloV1Block sdk.Int `json:"yoloV1_block" yaml:"yoloV1_block"` // YOLO v1: https://github.com/ethereum/EIPs/pull/2657 (Ephemeral testnet)
```

Comment:
this should be `YoloV2Block`

Reply:
see https://github.com/ethereum/go-ethereum/compare/v1.9.21...v1.9.24

Reply:
will re-add and update!